app/views/sessions/new.html.erb

```diff
@@ -7,12 +7,12 @@
             <h5 class="card-title text-center">Sign In</h5>
               <%= form_tag '/login', method: :post, :class => "form-signin" do %>
               <div class="form-label-group">
+                <%= text_field_tag :email, nil, {class: 'form-control'}  %>
```

Comment:
so simple and yet so hard to figure out! The forms no longer have weird orange box and the placeholder doesn't get in the way of writing.

Reply:
Way to fix this bug!

Reply:
is it just this 'form-control' class that did the trick?

Reply:
form-class all the way! Also the label and text field needed to be switched for each one for some reason.